app/Notifications/NewRequestUnclaim.php

```diff
@@ -42,6 +43,24 @@ public function via(object $notifiable): array
         return ['database'];
     }
 
+    /**
+     * Determine if the notification should be sent.
+     */
+    public function shouldSend(User $notifiable): bool
+    {
```

Comment:
Should probably not notify the requester if the unclaimer is the same user

Reply:
Added self check.